### ISSUE TITLE
Format option needed

### ISSUE THREAD

**Issue by BenLeroy** [iamkun/dayjs#330]
In momentJS, it's possible to specify the input date format like :

```
moment('26/05/1986 00:00:00', 'DD/MM/YYYY HH:mm:ss').format("YYYY-MM-DD HH:mm:ss")
```

I may have misread the docs but I didn't found an equivalent.

Actually if I do :
```
dayjs('26/05/1986 00:00:00').format("YYYY-MM-DD HH:mm:ss")
// outputs "1986-26-05 00:00:00"
```

**Comment by Imperat** [comment]
Hello, @BenLeroy. You are welcome in discussion here: https://github.com/iamkun/dayjs/issues/136

**Comment by iamkun** [comment]
:tada: CustomParseFormat plugin is supported and included in version 1.8.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/dayjs)
- [GitHub release](https://github.com/iamkun/dayjs/releases/tag/v1.8.0)
